File: 0268-missing-number/0268-missing-number.c
int missingNumber(int* nums, int numsSize) {
    int num;
    for(int i=0;i<numsSize;i++){
       for(int j=i+1;j<numsSize;j++){
        if(nums[i]>nums[j]){
            int temp=nums[i];
            nums[i]=nums[j];
            nums[j]=temp;
        }
       }
    }
     num=0;
    for(int i=0;i<numsSize;i++,num++){
        if(nums[i]!=num)
        break;
    }
    return num;
}

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length,actualsum=0;
        int exceptsum = (n*(n+1))/2;

        for(int num:nums){
            actualsum += num;
        }   

        return exceptsum - actualsum;
    }
}